entrypoint/upgradecluster/upgradenodereplacement_test.go

```diff
@@ -80,6 +79,10 @@ var _ = Describe("Test:", func() {
 			testcase.TestServiceLoadBalancer(false, true)
 		})
 	}
+
+	It("Deletes nodes created by aws cli", func() {
+		testcase.DeleteAWSInstance(cluster)
```

Comment:
should live on shared and not also be used as a It spec, since its not a actual test or feat